File: isem-finanzas/common/backend/database/Models/Modelos/AuditoriaModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Models.Modelos
{
    public class AuditoriaModel
    {
        public int? id_auditoria { get; set; }
        public int? ejercicio { get; set; }
        public string fecha_recepcion { get; set; }
        public string fecha_cumplimiento { get; set; }
        public string inicio_revision { get; set; }
        public string fin_revision { get; set; }
        public int? id_fiscal { get; set; }
        public int? id_area_remitente { get; set; }
        public int? id_remitente { get; set; }
        public string folio { get; set; }
        public bool? activo { get; set; }
        public int? id_tipo_recepcion { get; set; }
        public int? id_estatus { get; set; }
        public string descripcion { get; set; }
        public string nombre_auditoria { get; set; }
        public string num_auditoria { get; set; }
        public string fiscal_nombre { get; set; }
        public string area_nombre { get; set; }
        public string remitente_nombre { get; set; }
        public string tipo_recepcion { get; set; }
        public string estatus_nombre { get; set; }
        [NotMapped] public int? id_usuario_busqueda { get; set; }
        [NotMapped] public int tipoOperacion { get; set; }
    }

}

File: isem-finanzas-web/src/app/@finanzas/@auditorias/models/auditoria.model.ts
import { RequerimientoModel } from './requerimiento.model';

export class AuditoriaModel {
  id_auditoria?: number;
  ejercicio?: number;
  fecha_recepcion?: string;
  fecha_cumplimiento?: string;
  inicio_revision?: string;
  fin_revision?: string;
  id_fiscal?: number;
  id_area_remitente?: number;
  id_remitente?: number;
  //num_oficio?: string;
  folio?: string;
  activo?: boolean;
  id_tipo_recepcion?: number;
  id_estatus?: number;
  descripcion?: string;
  nombre_auditoria?: string;
  num_auditoria?: string;
  fiscal_nombre?: string;
  area_nombre?: string;
  remitente_nombre?: string;
  tipo_recepcion?: string;
  estatus_nombre?: string;
  id_usuario_busqueda?: number;
  dias_restantes?: number;
  //lista_requerimientos?: string;
}
